cuda/matrix/dense_kernels.cu

```diff
@@ -848,6 +848,42 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(
     GKO_DECLARE_DENSE_CALCULATE_MAX_NNZ_PER_ROW_KERNEL);
 
 
+namespace kernel {
+
+
+template <typename ValueType>
+__global__ __launch_bounds__(default_block_size) void count_nnz_per_row(
+    size_type num_rows, size_type num_cols, size_type stride,
+    const ValueType *__restrict__ work, size_type *__restrict__ result)
```

Comment:
Is this in here because of a merge?
I think below, it is already replaced by the `prefix_sum` kernels, so I don't think this kernel is used anywhere.